OpenRA.Mods.Common/Widgets/Logic/ModBrowserLogic.cs

```diff
@@ -82,17 +82,19 @@ public ModBrowserLogic(Widget widget)
 			{
 				try
 				{
-					using (var preview = new Bitmap(Platform.ResolvePath(ModMetadata.CandidateModPaths[mod.Id], "preview.png")))
-						if (preview.Width == 296 && preview.Height == 196)
-							previews.Add(mod.Id, sheetBuilder.Add(preview));
+					using (var stream = ModMetadata.AllMods[mod.Id].Package.GetStream("preview.png"))
```

Comment:
Actually it doesn't change anything in that regard. The mod runs (this can't break it even if it wanted to), and the mod's logo and preview images are still loaded (test case - d2k, as I already set it up to be ran from the Support dir on bleed).
So we're good here.